pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -75,7 +79,7 @@ func (c *kvEventToRowConsumer) topicForEvent(eventMeta cdcevent.Metadata) (Topic
 			return topic, nil
 		}
 	}
-	for _, s := range c.details.TargetSpecifications {
+	for _, s := range AllTargets(c.details) {
```

Comment:
Any chance to add a test for this? 
A direct call to topicForEvent with 2 protos -- 1 old 1 new.
You don't even need to construct full "consumer"-- just initialize "details" field.

Reply:
I made an attempt (code here: https://gist.github.com/HonoreDB/1386d17dedbd4150702f29495a2a6b1f) but am missing some kind of cleanup step and am probably not going to have time to figure it out tonight, have to wake up at 5 AM tomorrow. 
```
panic: rangefeed-system-monitor: unexpected 20480 leftover bytes

goroutine 940 [running]:
github.com/cockroachdb/cockroach/pkg/util/log/logcrash.ReportOrPanic({0x9e77748, 0xc00241ffb0}, 0xc00120e000, {0x88753dc, 0xc002c24800}, {0xc001df7730, 0x47c7c40, 0xc001df7718})
	github.com/cockroachdb/cockroach/pkg/util/log/logcrash/crash_reporting.go:376 +0x1af
github.com/cockroachdb/cockroach/pkg/util/mon.(*BytesMonitor).doStop(0xc000ada600, {0x9e77748, 0xc00241ffb0}, 0x1)
	github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go:426 +0x22a
github.com/cockroachdb/cockroach/pkg/util/mon.(*BytesMonitor).Stop(...)
	github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go:406
github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*BudgetFactory).Stop(0xc00125c480, {0x9e77748, 0xc00241ffb0})
	github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/budget.go:356 +0x45
```